src/main/java/com/openshift/internal/client/ApplicationSSHSession.java

```diff
@@ -393,7 +393,7 @@ protected InputStream execCommand(final String command, InputStream forStdIn,
 					"Could no execute remote ssh command \"{0}\" on application {1}",
 					command, application.getName());
 		} catch (IOException e) {
-			if (channel != null && channel.isConnected()) {
+			if (channel.isConnected()) {
```

Comment:
What is guaranteeing this? It's ensured at compile time that this was declared, but no guarantee that it was initialized